### PR TITLE
feat: updated major versions for frontend-platform & paragon

### PR DESCRIPTION
**Pull request by BilalQamar95**
### Description

- Replaced packages `@edx/paragon` & `@edx/frontend-build` to use openedx namespace. Updated imports & respective packages (Part of https://github.com/openedx/axim-engineering/issues/23)
- Major version upgrades, Updating `frontend-build` to v14 & `frontend-platform` to v8

### CODE REVIEW

**Comment by BilalQamar95** [comment]
@Ali-D-Akbar can you please merge it as well, I don't have the permission for it

**Comment by BilalQamar95** [comment]
> @Ali-D-Akbar can you please merge it as well, I don't have the permission for it

Hey @Ali-D-Akbar, are there any blockers on merging this?

**Comment by Ali-D-Akbar** [comment]
@BilalQamar95 Hi there, sorry about the delay. This PR requires some good local testing. We'll hopefully get this done this week and let you know if everything is okay and merge it. Thanks.

**Comment by BilalQamar95** [comment]
> @BilalQamar95 Hi there, sorry about the delay. This PR requires some good local testing. We'll hopefully get this done this week and let you know if everything is okay and merge it. Thanks.

@Ali-D-Akbar Any updates on when this is going to get merged?

**Review by AfaqShuaib09 — APPROVED** [review]
Tested on local, Changes LGTM 👍🏻 , Good to go

**Comment by BilalQamar95** [comment]
> @Ali-D-Akbar can you please merge it as well, I don't have the permission for it

@AfaqShuaib09 can you please merge it?

File: package.json [hunk 1]
```diff
@@ -16,17 +16,17 @@
...
   "license": "AGPL-3.0",
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
-    "@edx/frontend-component-footer": "npm:@edx/frontend-component-footer-edx@^6.4.0", 
-    "@edx/frontend-platform": "5.6.1",
+    "@edx/frontend-component-footer": "npm:@edx/frontend-component-footer-edx@^7.1.0",
+    "@edx/frontend-platform": "^8.0.0",
     "@edx/openedx-atlas": "^0.6.0",
-    "@edx/paragon": "^20.44.0",
+    "@openedx/paragon": "^21.11.3",
```
Comment (zawan-ila) [inline package.json:29]: Why are we moving some packages from the edx namespace to openedx?

**Review by zawan-ila — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (BilalQamar95) [reply package.json:29]: You can view [axim-engineering#23](https://github.com/openedx/axim-engineering/issues/23) for details. I have also updated the description

**Pull request merged**